packages/js/components/src/date-time-picker-control/date-time-picker-control.tsx

```diff
@@ -37,7 +37,7 @@ export type DateTimePickerControlProps = {
 export const DateTimePickerControl: React.FC< DateTimePickerControlProps > = ( {
 	currentDate,
 	is12Hour = true,
-	dateTimeFormat = is12Hour ? 'MM/DD/YYYY h:mm a' : 'MM/DD/YYYY H:MM',
+	dateTimeFormat = is12Hour ? 'MM/DD/YYYY h:mm a' : 'MM/DD/YYYY H:mm',
```

Comment:
Maybe these date formats should be consts. If we are going to use them in other place like for validating the control then we will have a source of truth to know which are the right formats.

Reply:
Good idea. I've pushed commits to export these consts and use them in the tests.